lib/sync.js

```diff
@@ -39,21 +41,19 @@ const postHeaders =  {
 function _authenticate(server, json, callback) {
     json.app_id = '';
     var options = {
-        url: server + 'auth',
+        url: server + '/auth',
         body: JSON.stringify(json),
         headers: postHeaders
     };
     post(options, function(error, response, body) {
         if (error) {
-            console.log(error);
```

Comment:
IMO plainly `console.log`-ing errors in a library looks very unprofessional to me, but I can bring back the `console.log` statements if you like.